routers/api/v1/api.go

```diff
@@ -1301,6 +1301,9 @@ func Routes() *web.Route {
 			m.Group("/actions/secrets", func() {
 				m.Get("", reqToken(), reqOrgOwnership(), org.ListActionsSecrets)
 				m.Post("", reqToken(), reqOrgOwnership(), bind(api.CreateSecretOption{}), org.CreateOrgSecret)
```

Comment:
There was bug in `CreateOrgSecret`: It doesn't validate the name (see actions.NameRegexMatch)

Reply:
fixed in 3a4cd0cfe